utils/converters.py

```diff
@@ -50,3 +50,16 @@ def DatetimeConverter(arg: str):
         return now + delta
     except (ValueError, KeyError) as e:
         raise commands.BadArgument('Invalid duration format.')
+
+class ListConverter(commands.Converter):
+    async def convert(self, ctx: commands.Context, argument: str) -> list[int] | str:
+        """
+        Checks if provided list can be separated and parsed.
+        """
+        if argument == '*':
+            return '*'
+
+        try: 
+            return list(map(int, re.split(';|,|; |, ', argument)))
```

Comment:
It might be natural to separate list entries with spaces as a message command; I would tweak the regex to accommodate this, I'm not sure if `re.split('\W+', argument)` will cause bugs but it could work.

Does something like `commands.Greedy` run into issues? I was going to suggest this or suggest using keyword-only arguments for the commands but I do not recall if they are supported with hybrid commands yet.

Reply:
`commands.Greedy` isn't really compatible with slash commands so that's out of the picture. I feel like adding spaces might cause more problems/accidental typos as spaces aren't as obvious as a `;` or `,` delimiter. What do you think?